src/main/frontend/handler/events.cljs

```diff
@@ -85,6 +96,20 @@
   (when-let [repo (get-local-repo)]
     (state/set-modal! (ask-permission repo))))
 
+
+
+(defmethod handle :after-db-restore [[_ repo]]
+  (println ":after-db-restore: " repo)
+  (let [conn* (conn/get-conn repo)
```

Comment:
oh, I forgot this one, `conn/get-conn` actually returns a db instead of a conn (which is an atom), it's a long story 😄 
usually, something that ends with a `*` identified itself as a ref, but it's not here.

Reply:
updated!